src/TunNetCom.SilkRoadErp.Sales.Api/Features/Products/GetProductByRef/GetProductByRefEndpoint.cs

```diff
@@ -0,0 +1,26 @@
+﻿
+using TunNetCom.SilkRoadErp.Sales.Contracts.Products;
+
+namespace TunNetCom.SilkRoadErp.Sales.Api.Features.Products.GetProductByRef
+{
+    public class GetProductByRefEndpoint : ICarterModule
+    {
+        public void AddRoutes(IEndpointRouteBuilder app)
+        {
+            app.MapGet("/Products/{refe}", async Task<Results<Ok<ProductResponse>, NotFound>> (IMediator mediator, string refe) =>
```

Comment:
{refe:string}
CancellationToken

Reply:
{refe:string} : make error
so i asked chatgpt and tell me that we don't need to explicitly specify type